src/ameba/ast/visitors/scope_visitor.cr

```diff
@@ -3,6 +3,21 @@ require "./base_visitor"
 module Ameba::AST
   # AST Visitor that traverses the source and constructs scopes.
   class ScopeVisitor < BaseVisitor
+    SUPER = "super"
```

Comment:
What is this constant for? Unlike Ruby (where strings are mutable), Crystal doesnt gain any benefit from extracting string literals to constants.

Reply:
Constant is created once across the application, literals (in a method) are created each time you call a method.

Reply:
We already discussed that https://github.com/veelenga/ameba/pull/50#issuecomment-381723855